### DISCUSSION TITLE
how to yAxis format?

### DISCUSSION THREAD

**Discussion by jhj0088** [nhn/tui.chart#727] [Q&A]
I want to change the data displayed on yAxis to the numeric type. ex) 1234 -> 1,234

**Comment by jwlee1108** [comment]
@jhj0088 

I think you can do it. Please refer to the link below.
https://github.com/nhn/tui.chart/blob/main/docs/ko/common-axes.md#formatter

**Reply by jwlee1108** [reply]
@jhj0088 It converted to the discussion.

**Reply by leehongtaek** [reply]
저도 Y 축만 이상하게 format이 안되더라고요..

`const options = {
				chart: { width: 780, height: 400, format : '1,000' },
				yAxis: [
					{ title: '사업비(억)',  pointOnColumn: false, chartType: 'line', scale: { stepSize: 2000 }
					},
					{ title: '건 수',  pointOnColumn: false, chartType: 'column', scale: { stepSize: 4 }
					},
				],`
